tests/roles/backend_services/templates/openstack_control_plane.j2

```diff
@@ -82,6 +82,15 @@ spec:
         secret: osp-secret
         replicas: 1
         storageRequest: 1Gi
+      # TODO(bogdando): iterate based on renamed_cells value in kustomization.yaml
```

Comment:
+1 on the TODO i think this is an important follow-up because the current state increases HW requirements for the test environment, if we deploy 3 cell DBs and MQs always.

Reply:
Yes, I have already prepared a follow up commit, yet don't want to submit it here to not retrigger all the testing including testprojects